tests/unit_tests/callbacks/tracers/test_langchain_v1.py

```diff
@@ -0,0 +1,684 @@
+"""Test Tracer classes."""
```

Comment:
think we should split this up into two files: one with tests for basetracer (which uses faketracer) and one which checks the conversion and uses the langchain_tracer_v1 fixture

Reply:
Sounds good